apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -195,6 +197,16 @@ type ASMIASTConfig struct {
 	Enabled *bool `json:"enabled,omitempty"`
 }
 
+// ProfilingFeatureConfig contains Continuous Profiler configuration.
+// Note that this will only affect pods where the Datadog client libraries are installed or APM Single Step Instrumentation is enabled.
+type ProfilingFeatureConfig struct {
+	// Enabled enables Continuous Profiler.
+	// Default: ""
+	// Valid values: "", "true", "false", "auto"
```

Comment:
You may want to use an enum here for schema validation to happen correctly

Reply:
Sorry there was not enough context to my request. The CRD is built using [Kubebuiler controller-gen CLI](https://book.kubebuilder.io/introduction) and it has a feature to directly add an [enum in the openapi schema validation](https://book.kubebuilder.io/reference/markers/crd-validation) done synchronously on the Manifest of the resource in question when applied. Adding this to the field and calling `make generate` again should do it:
```suggestion
	// Valid values: "", "true", "false", "auto"
	// +kubebuilder:validation:Enum=true;false;auto;
	// +optional
	Enabled *string `json:enabled,omitempty`
```